### DISCUSSION TITLE
项目中的某些类别，比如说商品，是推荐使用struct来构建还是class呢？

### DISCUSSION THREAD

**Discussion by tang-mouren** [liusy58/NJU_AdvancedProgramming_SP22#3] [Q&A] [answered]

**Comment by liusy58** [comment, marked as answer]
两者并没有很大的区别，C++为了向下兼容仍然保留了struct，当然还是更推荐class.

**Comment by RicoloveFeng** [comment]
在 C++ 里面 struct 就是一种特殊的 class。如果你想像 C 那样弄一个 struct，实际上让 class 的成员变量全 public 就能有同样的效果。重点在于怎么设计各个类之间的分工，而不至于弄出一个「用 class 包起来的 main 函数」这样的东西。
